src/bin/cobalt/new.rs

```diff
@@ -409,8 +419,26 @@ pub fn publish_document(file: &path::Path) -> Result<()> {
     let doc =
         cobalt_model::DocumentBuilder::<cobalt_model::FrontmatterBuilder>::new(front, content);
     let doc = doc.to_string();
-
     cobalt_model::files::write_document_file(doc, file)?;
 
+    if config.posts.add_date_prefix {
```

Comment:
From what little I remember of this code path, don't we need to check if the user is publishing a post?

Reply:
Can we publish something else than a post?

Reply:
Nothing in the code currently blocks it.  You could add a check or that or support both.

Reply:
To detect if it's a post, am I right if I say that I need to check the file path and compare it against the configuration `posts.dir"?